devices/Bq2579x/Bq2579x.cs

```diff
@@ -261,6 +268,58 @@ public class Bq2579x : IDisposable
         [Property]
         public ThermalShutdownThreshold ThermalShutdownThreshold { get => GetThermalShutdownThreshold(); set => SetThermalShutdownThreshold(value); }
 
+        /// <summary>
+        /// Gets or sets charge voltage for JEITA high temperature range (TWARN � THOT). 
+        /// </summary>
+        [Property]
+        public ChargeVoltage ChargeVoltageHighTempRange { get => GetChargeVoltage(); set => SetChargeVoltage(value); }
+
+        /// <summary>
+        /// Gets or sets charge current for JEITA high temperature range (TWARN � THOT). 
+        /// </summary>
+        [Property]
+        public ChargeCurrent ChargeCurrentHighTempRange { get => GetChargeCurrentHighTempRange(); set => SetChargeCurrentHighTempRange(value); }
+
+        /// <summary>
+        /// Gets or sets charge current for JEITA low temperature range (TCOLD � TCOOL). 
```

Comment:
and here: 
![image](https://github.com/nanoframework/nanoFramework.IoT.Device/assets/8145578/53a54b76-0e6f-4321-8637-ea78e6b92f69)